src/voters/voterFile/schemas/ScheduleOutreachCampaign.schema.ts

```diff
@@ -25,7 +26,22 @@ export class ScheduleOutreachCampaignSchema extends createZodDto(
         .strict(),
     ),
     script: z.string(),
-    date: z.string().date(),
+    date: z
+      .string()
+      .date()
+      .refine(
+        (date) => {
+          const selectedDate = startOfDay(parseISO(date))
+          const minDate = startOfDay(addDays(new Date(), 3))
+          return (
+            isAfter(selectedDate, minDate) ||
+            selectedDate.getTime() === minDate.getTime()
+          )
+        },
+        {
+          message: 'Date must be at least 72 hours from now',
+        },
+      ),
```

Comment:
I think we may need to base this off of the current "now" date from the client?

Our API is running in Oregon, there could be some discrepancy here if someone is making a request from another time zone that is closer to the day boundary like the East Coast or Hawaii, right? 🤔